source/operation.ts

```diff
@@ -4,6 +4,8 @@
  * @namespace operation
  */
 
+export interface Operation {}
```

Comment:
Expand this one to add `action` and add separate interfaces for the different operation types

Reply:
fixed!